File: exemplos/AnoBissexto.java
package exemplos;
public class AnoBissexto {
    public static void main(String[] args) {
        int ano = 2024;

        boolean ehBissexto = (ano % 4 == 0 && ano % 100 != 0) || (ano % 400 == 0);

        if (ehBissexto) {
            System.out.println("O ano é bissexto.");
        } else {
            System.out.println("O ano não é bissexto.");
        }
    }
}

File: exemplos/AnoBissexto.py
ano = 2024

ehBissexto = (ano % 4 == 0 and ano % 100 != 0) or (ano % 400 == 0)

if ehBissexto:
    print("O ano é bissexto.")
else:   
    print("O ano não é bissexto.")
